config/newrelic.yml

```diff
@@ -13,7 +13,7 @@ production:
   error_collector:
     enabled: true
     capture_source: true
-    ignore_errors: "ActionController::RoutingError,ActionController::BadRequest,ActionDispatch::Http::MimeNegotiation::InvalidType"
+    ignore_errors: "ActionController::RoutingError,ActionController::BadRequest,ActionDispatch::Http::MimeNegotiation::InvalidType,GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError"
```

Comment:
I think we'll need a way to not raise during the job itself, otherwise our alarms will go off

Reply:
Ok, my thinking is deploy this and let  it sit overnight and enqueue those daily jobs that should collide, and then check the logs?